addons/parachute/functions/fnc_storeParachute.sqf

```diff
@@ -17,7 +17,7 @@
 #include "script_component.hpp"
 
 params ["_unit", "_gear"];
-private _backpack = _gear select 6;
+private _backpack = _gear param [6, ""];
```

Comment:
Isnt the backpack:
`_gear param [5, []] param [0, ""]` ???

Or is this using FUNC(getAllGear) ?